api/api.go

```diff
@@ -40,7 +40,10 @@ import (
 //	@tag.name			Wallet
 //	@tag.description	Operations for the wallets managed backend side
 
-//	@BasePath					https://api-dev.vocdoni.net/v2/
+//	@schemes	https
+//	@host		api-dev.vocdoni.net
+//	@BasePath	/v2
```

Comment:
Should be `basePath` instead of `BasePath`.

Reply:
Ok now I see it's something (terrible) from swaggo https://github.com/swaggo/swag#declarative-comments-format